kontrakter/dom/begjaeringDom/arbeidsversjon/begjaeringDom.schema.json

```diff
@@ -0,0 +1,1297 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://politiet.no/dom/arbeidsversjon/begjaeringDom",
+  "description": "Skjema for begjæring av dom til tingretten første omgang tilståelsesdom, JSON Schema versjon 2020-12",
+
+  "type": "object",
+  "properties": {
+    "forsendelse": {
+      "$ref": "#/definitions/forsendelse",
+      "description": "avsender og mottaker"
+    },
+    "hovedStraffesaksnummer": {
+      "$ref": "#/definitions/straffesaksnummer",
+      "description": "BL Saksbehandlersak er hovedsaken i et sakskompleks (som er 1 eller flere straffesaker)"
+    },
+    "begjaeringVaretekt": {
```

Comment:
Hvorfor trenger vi denne i begjæringen om dom. Er det for å eventuelt registrere straffefradrag for dager i varetekt?